### DISCUSSION TITLE
Live reload is not working on deno 1.32.4

### DISCUSSION THREAD

**Discussion by maxchang3** [lumeland/lume#412] [General]
I’m trying to change something to trigger live reload, but it doesn’t work. I also found that nothing was sent to the WebSocket client. 

By troubleshooting, I found out it’s due to deno itself:

https://github.com/denoland/deno/issues/18715

https://github.com/denoland/deno/issues/18719

maybe the temporary solution: upgrade to canary version or downgrade to 1.32.3.

**Comment by oscarotero** [comment]
Oh, thanks for mentioning this. I had the same problem yesterday and didn't know the origin of the issue.
I hope they release a new Deno version soon.
